test_conformance/common/vulkan_wrapper/vulkan_wrapper.cpp

```diff
@@ -130,6 +152,32 @@ VulkanInstance::VulkanInstance(): m_vkInstance(VK_NULL_HANDLE)
     VK_GET_NULL_INSTANCE_PROC_ADDR(vkCreateInstance);
 #undef VK_GET_NULL_INSTANCE_PROC_ADDR
 
+    if (m_useValidationLayers)
+    {
+        uint32_t layerCount = 0;
+        vkEnumerateInstanceLayerProperties(&layerCount, nullptr);
+
+        std::vector<VkLayerProperties> layers(layerCount);
+        vkEnumerateInstanceLayerProperties(&layerCount, layers.data());
+
+        for (auto it = m_validationLayers.begin();
+             it != m_validationLayers.end();)
+        {
+            bool found = false;
+            for (const auto &layerProps : layers)
+                if (strcmp(*it, layerProps.layerName) == 0)
+                {
+                    found = true;
+                    break;
+                }
+            if (!found)
+                it = m_validationLayers.erase(it);
```

Comment:
Nice-to-have: We may want to print a message here if a validation layer is not found vs. silently removing it.

Reply:
Done